assets/styles/elements/menu-item-link.css

```diff
@@ -0,0 +1,28 @@
+.menu-item__link {
```

Comment:
My idea of the elements folder was to override styles of standard elements here (buttons, links, headings, etc.). So I would move this either to the root styles folder or create a common partial for that kind of menu.

From what I see we have two variants.

1. <img width="1206" alt="image" src="https://github.com/user-attachments/assets/5086c350-ae89-4d51-ab23-7c6de36124d2">
2. <img width="1134" alt="image" src="https://github.com/user-attachments/assets/677eebeb-d5f3-4633-bc1d-39f89fdabaa8">

Where the main differences seem to be the header size (I would prefer the smaller one) and the blue tags (which can be optional).

Reply:
@lukket Yes, header logic is different. time logic is different, tag logic is different - only container is common, I don't see the reason to create a common partial with a lot of if-else

Reply:
You're right, I just saw it from a design perspective and forgot the date stuff.

Reply:
Still, please consider moving it out of the elements folder. If it stays there, it would imply, from my current understanding, that we should apply this style to every menu item.

Reply:
moved